src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Components.ServiceWorkerAssetsManifest.targets

```diff
@@ -121,7 +121,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     </PropertyGroup>
 
     <ItemGroup>
-      <_ServiceWorkerIntermediatePublishFile Include="%(ServiceWorker.IntermediateOutputPath)serviceworkers\%(FileName).publish%(Extension)">
+      <_ServiceWorkerIntermediatePublishFile Include="$(IntermediateOutputPath)serviceworkers\%(FileName).publish%(Extension)">
```

Comment:
I'm not understanding what is being selecting/included here. By removing `%(ServiceWorker.` where are the items coming from?

Reply:
It constructs the path that later gets written to:

![image](https://user-images.githubusercontent.com/174281/85614050-5f6d8880-b60f-11ea-9bcc-9ecd308da19d.png)

It is a bit awkward that the output is the identity, but that's how this target was previously written